app/models/policy_host.rb

```diff
@@ -9,8 +9,19 @@ class PolicyHost < ApplicationRecord
   validates :host, presence: true, on: :create
   validates :host_id, presence: true, uniqueness: { scope: :policy }
 
+  scope :supported_host_ids, lambda { |policy_id, host_ids|
```

Comment:
You'd need a `supported_os_versions` instead, maybe `supported_os_minors` is enough...